packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -59,16 +85,17 @@ export default  class FileInput extends UIPlugin {
     // ___Why not use value="" on <input/> instead?
     //    Because if we use that method of clearing the input,
     //    Chrome will not trigger change if we drop the same file twice (Issue #768).
+    // @ts-expect-error yes
     event.target.value = null // eslint-disable-line no-param-reassign
   }
 
-  handleClick () {
-    this.input.click()
+  handleClick (): void {
+    this.input?.click()
   }
 
-  render () {
+  render (): ComponentChild {
```

Comment:
```suggestion
  render (): JSX.Element {
```

Reply:
I think ComponentChild is correct. afaik it's similar to React's ReactNode, in that it's anything that `render` can return. JSX.Element is **only** JSX elements, nothing else. FWIW the super class (UIPlugin.ts) also returns type ComponentChild

Reply:
from `preact`:
```ts
type ComponentChild =
	| VNode<any>
	| object
	| string
	| number
	| bigint
	| boolean
	| null
	| undefined;
```

the reason why it's a better type is because it allows the `render` function to return everything that is valid for a `render` function to return. e.g. sometimes we might to do something like `if (data == null) return null`, but with the JSX.Element return type, that would not be possible